clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -63,11 +63,11 @@ const (
 	countFastQuery = "sum(increase(apiserver_request_duration_seconds_bucket{%v}[%v])) by (resource, subresource, scope, verb)"
 
 	// exclude all buckets of 1s and shorter
-	filterGetAndMutating = `verb!~"WATCH|WATCHLIST|PROXY|CONNECT|LIST", le="1"`
+	filterGetAndMutating = `resource!="events", verb!~"WATCH|WATCHLIST|PROXY|CONNECT|LIST", le="1"`
```

Comment:
Why you're readding events?

Reply:
readding?
I've removed events, because they were not counted in 1.18. Check line https://github.com/kubernetes/perf-tests/pull/1602/files#diff-d4933bdf320dd42179e6f8ef4c50c4ff7c2aa5a5b35f179278e5adbd345ae8ebR47

Reply:
This was the reason why SlowCount was negative in our tests. SLO for events was enabled only in 1.19.

Reply:
I mean readding excluding events.

But my question is - why this can reasult in negative slo count? events should be orthogonal

Reply:
This is because line 47:
```
filters = `resource!="events", verb!~"WATCH|WATCHLIST|PROXY|CONNECT"`
``` 
needs to match this line. 

We were not counting events there but counting here, hence negative count

Reply:
Slow count is computed as WholeCount - FastCount. for WholeCount (line I posted above) we exclude events. So when object in golang is created default value is 0. Then we subtract event from FastCount and that's the reason why it's negative.